#request constants
int8 DISSECTION=1
int8 DISTANT=2
int8 GRABBING=3
int8 RETRACTION=4
#request fields
int8 target_type
---
#response constants
int8 DP=3
int8 GOAL=4
#response fields
int8 position_type
geometry_msgs/Pose pose
